### PR TITLE
[faro collector] add missing struct fields so that payload.meta.browser fields are not lost

### LINKED ISSUE #1832: [faro collector] payload.meta.browser fields are not decoded and forwarded.
### What's wrong?

a few fields are missing in the struct which is used to read the meta.browser object that faro sends, which results in that information being lost. https://github.com/grafana/faro-web-sdk/blob/v1.10.2/packages/core/src/metas/types.ts#L60-L70

### Steps to reproduce

send logs to an alloy faro collector endpoint using the faro sdk.

### System information

_No response_

### Software version

_No response_

### Configuration

_No response_

### Logs

_No response_

### PR DESCRIPTION
**Pull request by elee1766**
#### PR Description

a few fields are missing in the struct which is used to read the meta.browser object that faro sends, which results in that information being lost. https://github.com/grafana/faro-web-sdk/blob/v1.10.2/packages/core/src/metas/types.ts#L60-L70

#### Which issue(s) this PR fixes

fixes #1832

#### Notes to the Reviewer

i did not do the work for properly parsing `NavigatorUABrandVersion[] | string`, since it's more work, and i mostly really just want viewportWidth and viewportHeight. since that field is already being ignored right now, it should be safe to not do until maybe someone else who needs it can add it. 

#### PR Checklist

- [x] CHANGELOG.md updated
- [ ] Documentation added
- [x] Tests updated
- [ ] Config converters updated

### CODE REVIEW

**Comment by elee1766** [comment]
i dont think i have to modify any config adapters or documentation? if i do please let me know

**Comment by wildum** [comment]
@t00mas do you have some context for this PR?

**Comment by t00mas** [comment]
No context, but LGTM

I'd like to have an issue created, so I can reference that in other components too, please.

File: CHANGELOG.md [hunk 1]
```diff
@@ -50,6 +50,7 @@ Main (unreleased)
 
 - Fix issue where `loki.source.kubernetes` took into account all labels, instead of specific logs labels. Resulting in duplication. (@mattdurham)
 
+- Fix an issue where some `faro.receiver` would drop multiple fields defined in `payload.meta.browser`, as fields were defined in the struct
```
Comment (wildum) [inline CHANGELOG.md:53]: nit: you can add your github @ if you want credit for it

**Review by wildum — APPROVED** [review]
LGTM, thanks!

**Comment by elee1766** [comment]
hi. sorry the context is i wanted the fields so i opened a PR.

i opened a basic issue.
#1832

**Pull request merged**